profiling/resnet_model.py

```diff
@@ -34,7 +34,7 @@
 from tensorflow.keras import layers as tf_python_keras_layers
 from tensorflow.keras import models
 from tensorflow.keras import regularizers
-from tensorflow_examples.profiling import imagenet_preprocessing_ineffecient_input_pipeline
+from examples.profiling import imagenet_preprocessing_ineffecient_input_pipeline
```

Comment:
This doesn't  look right.

```
$> pip install git+https://github.com/tensorflow/examples
$> python
>>> import tensorflow_examples
```


maybe something's missing from the setup.py

Reply:
Nope.

The problem is that this "profiling" directory is not in  the "tensorflow_examples" directory of the repository. We may need to fix that i n the copybara script.